src/graph/rg_matrix/rg_pending.c

```diff
@@ -0,0 +1,50 @@
+/*
+* Copyright 2018-2021 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#include "RG.h"
+#include "rg_matrix.h"
+
+GrB_Info RG_Matrix_pending
+(
+	const RG_Matrix C,              // matrix to query
+	bool *pending                   // are there any pending operations
+) {
+	ASSERT(C       != NULL);
+	ASSERT(pending != NULL);
+
+	GrB_Info    info;
+	bool        p        =  false;
+	bool        res      =  false;
+	GrB_Matrix  M        =  RG_MATRIX_M(C);
+	GrB_Matrix  DP       =  RG_MATRIX_DELTA_PLUS(C);
+	GrB_Matrix  DM       =  RG_MATRIX_DELTA_MINUS(C);
+
+	if(RG_MATRIX_MAINTAIN_TRANSPOSE(C)) {
+		info = RG_Matrix_pending(C->transposed, &res);
```

Comment:
if transposed matrix has pending then no need to check non transposed

Reply:
👍